src/gui/tray/usermodel.cpp

```diff
@@ -174,7 +174,7 @@ void User::showDesktopNotification(const Activity &activity)
 
 void User::showDesktopNotification(const ActivityList &activityList)
 {
-    const auto subject = tr("%1 notifications").arg(activityList.count());
+    const auto subject = tr("%n notification(s)", "", activityList.count());
```

Comment:
Can also just pass a `nullptr` for the disambiguation (it's the default value)
```suggestion
    const auto subject = tr("%n notifications", nullptr, activityList.count());
```